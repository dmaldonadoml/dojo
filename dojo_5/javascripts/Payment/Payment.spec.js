const Payment = require('./Payment')
const Congrats = require('../congrats')

const efecty = 'efecty'
const visa = 'visa'

describe('Payment', () => {
  
  it('Interface', () => {
    const payment = Payment()
    expect(payment).toMatchSnapshot()
  })  
  describe('congrats', () => {
    
    it('Efecty', () => {
      const Efecty = Payment().createType(efecty)
      expect(Efecty.congrat()).toMatchSnapshot()
    })
    it('visa', () => {
      const Visa = Payment().createType(visa)
      expect(Visa.congrat()).toMatchSnapshot()
    })
  })  
})