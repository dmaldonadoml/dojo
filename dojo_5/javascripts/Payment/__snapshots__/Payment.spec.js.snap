// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Payment Interface 1`] = `
Object {
  "congrat": [Function],
  "createType": [Function],
  "type": undefined,
  "visit": [Function],
}
`;

exports[`Payment congrats Efecty 1`] = `
Object {
  "heading": "¡Apúrate a pagar!",
  "status": "success",
  "substatus": null,
  "title": "Paga \${price} en \${paymentMethodName} para reservar tu compra",
}
`;

exports[`Payment congrats visa 1`] = `
Object {
  "heading": "¡Excelente compra!",
  "status": "success",
  "substatus": null,
  "title": "El jueves 2017-05-18T00:00:00.000-05:00. te llegará el producto",
}
`;
